includes/testing-library.sh

```diff
@@ -86,6 +99,28 @@ get_composer_bin_dir() {
     echo ${composer_bin_dir}
 }
 
+get_project_type_directory() {
```

Comment:
We now have one function suffixed with "_dir" and one with "_directory", we should keep that consistent. I like your version better, so let's rename get_composer_bin_dir to get_composer_bin_directory.